### ISSUE TITLE
accuracy on casia training dataset

### ISSUE THREAD

**Issue by auroua** [wy1iu/sphereface#60]
I am using the sphereface code implemented by pytorch in the third-party re-implementation lists.
After 20 epochs, I only get `93.3490%` accuracy on training dataset. Does this correct?
This model get  `98.85%` accuracy on lfw.

**Comment by jjennings955** [comment]
I have the same question. 

@auroua is your 93.3490% on withheld validation data from CASIA?

**Comment by auroua** [comment]
@jjennings955 not on validate dataset. It's on training dataset.

**Comment by wy1iu** [comment]
I am not sure about the pytorch version, but my guess is that when you compute the accuracy, it uses the difficult decision rule (because m=4 makes the classification more difficult). You will need to use m=1 to compute the accuracy.

**Comment by auroua** [comment]
@wy1iu Thanks
